Sources/TypeformUI/Components/IntermittentChoiceButtonStyle.swift

```diff
@@ -27,15 +27,35 @@ struct IntermittentChoiceButtonStyle: ButtonStyle {
     }
     
     private var backgroundColor: Color {
-        selected ? settings.interaction.selectedBackgroundColor : settings.interaction.unselectedBackgroundColor
+        if style == nil {
+            return selected ? settings.rating.selectedBackgroundColor : settings.rating.unselectedBackgroundColor
+        } else {
+            return selected ? settings.interaction.selectedBackgroundColor : settings.interaction.unselectedBackgroundColor
+        }
     }
     
     private var strokeColor: Color {
-        selected ? settings.interaction.selectedStrokeColor : settings.interaction.unselectedStrokeColor
+        if style == nil {
+            return selected ? settings.rating.selectedStrokeColor : settings.rating.unselectedStrokeColor
+        } else {
+            return selected ? settings.interaction.selectedStrokeColor : settings.interaction.unselectedStrokeColor
+        }
     }
     
     private var strokeWidth: Double {
-        selected ? settings.interaction.selectedStrokeWidth : settings.interaction.unselectedStrokeWidth
+        if style == .none {
```

Comment:
@richardpiazza Just want to confirm...Is there a difference here between `style == nil` and `style == .none`.  Is this correct behavior for the `strokeWidth` and `foregroundColor`?

Reply:
Great question. Nope, `style == nil` and `style == .none` are functionally equivalent here. (`.none` being the `Optional<Style>` case representing no value. I wrote it that way to try and express the _silent third_ case of !radio and !checkbox.
<img width="268" alt="Screenshot 2024-03-12 at 8 07 34 AM" src="https://github.com/Nice-Healthcare/typeform-swift/assets/1744463/ca52826d-565a-4752-8513-ac8d99c4810a">